htsinfer/get_library_source.py

```diff
@@ -82,34 +82,11 @@ def evaluate(self) -> ResultsSource:
                 self.tax_id,
                 self.transcripts_file
             )
+            source.file_1.short_name = src_name
 
-            if src_name is not None:
-                source.file_1.short_name = src_name
-
-                if self.paths[1] is not None:
-                    source.file_2.taxon_id = self.tax_id
-                    source.file_2.short_name = source.file_1.short_name
-
-            else:
-                LOGGER.warning(
-                    f"Taxon ID '{self.tax_id}' not found in "
-                    "organism dictionary, inferring source organism..."
-                )
-                index = self.create_kallisto_index()
-                library_source = self.get_source(
-                    fastq=self.paths[0],
```

Comment:
But are you even still calling `.get_source_name()` now? I don't see it anywhere

Reply:
Please see my other comment. If tax-id parameter is given, it's assigned as the `source.file_1.taxon_id` and the `.get_source_name()` is called to get `source.file_1.short_name` (and same for `source.file_2`). If tax-id is not set, then both params are inferred.